src/main/java/com/github/maleksandrowicz93/cqrsdemo/student/exception/ErrorMessage.java

```diff
@@ -0,0 +1,18 @@
+package com.github.maleksandrowicz93.cqrsdemo.student.exception;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+
+@Getter
+@RequiredArgsConstructor
+public enum ErrorMessage {
+
+    STUDENT_ALREADY_EXISTS("Cannot add student because already exists.", HttpStatus.BAD_REQUEST),
+    INVALID_CREDENTIALS("Username or password is invalid.", HttpStatus.BAD_REQUEST),
+    STUDENT_NOT_FOUND("Student not found.", HttpStatus.NOT_FOUND),
+    UNKNOWN_ERROR("Unknown error.", HttpStatus.INTERNAL_SERVER_ERROR);
```

Comment:
tutaj warto byłoby zostawić TODOsa na obsługę i18n - zamiast stałych wartości klucze i wartości pobierane z resource'ów dynamicznie

Reply:
i18n wzięta pod uwagę - chciałem, żeby translation key były nazwy enumów, być może brakuje javadoców, w których pewnie bym to opisał :)